src/EditorFeatures/TestUtilities/Workspaces/TestWorkspaceFixture.cs

```diff
@@ -62,9 +62,20 @@ public TestWorkspaceFixture()
 
         public void Dispose()
         {
-            if (_workspace != null)
+            if (_workspace is null)
+                return;
```

Comment:
if this hits, is that a test issue (double disposing?)  should we assert then?

Reply:
It's not a problem.

* `ReferenceCountedDisposable<T>` ensures single-dispose
* A test fixture can be created without actually creating a workspace within it